Representable/representable
# Hmong css fixes

**changes**
fixes various frontend bugs for hmong translations:
- comm info button on mapping survey page being too long
- next and previous buttons on survey page too long and overlapping
- comm info going off the card on submission page
- demographics and data layers tabs bleeding into each other on submission page

** to test**
- python manage.py runserver
- switch language to hmong
- check above spots ^^ for desired behavior
